contracts/src/payments/SuccinctFeeVault.sol

```diff
@@ -168,4 +157,8 @@ contract SuccinctFeeVault is IFeeVault, TimelockedUpgradeable {
 
         emit Collected(_to, _token, _amount);
     }
+
+    /// @dev This empty reserved space to add new variables without shifting down storage.
+    ///      See: https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+    uint256[50] private __gap;
```

Comment:
Forgot to do this previously, but it's fine since we go from:
```
| Name             | Type                                                         | Slot | Offset | Bytes | Contract                                           |
|------------------|--------------------------------------------------------------|------|--------|-------|----------------------------------------------------|
| _initialized     | uint8                                                        | 0    | 0      | 1     | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| _initializing    | bool                                                         | 0    | 1      | 1     | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 1    | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 51   | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 101  | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 151  | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| _roles           | mapping(bytes32 => struct AccessControlUpgradeable.RoleData) | 201  | 0      | 32    | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[49]                                                  | 202  | 0      | 1568  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| balances         | mapping(address => mapping(address => uint256))              | 251  | 0      | 32    | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| allowedDeductors | mapping(address => bool)                                     | 252  | 0      | 32    | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |

```

To:
```
| Name             | Type                                                         | Slot | Offset | Bytes | Contract                                           |
|------------------|--------------------------------------------------------------|------|--------|-------|----------------------------------------------------|
| _initialized     | uint8                                                        | 0    | 0      | 1     | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| _initializing    | bool                                                         | 0    | 1      | 1     | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 1    | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 51   | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 101  | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 151  | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| _roles           | mapping(bytes32 => struct AccessControlUpgradeable.RoleData) | 201  | 0      | 32    | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[49]                                                  | 202  | 0      | 1568  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| balances         | mapping(address => mapping(address => uint256))              | 251  | 0      | 32    | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| allowedDeductors | mapping(address => bool)                                     | 252  | 0      | 32    | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |
| __gap            | uint256[50]                                                  | 253  | 0      | 1600  | src/payments/SuccinctFeeVault.sol:SuccinctFeeVault |

```